express-pipeline/dist/styles.js

```diff
@@ -1,13 +1,13 @@
 "use strict";
 
 Object.defineProperty(exports, "__esModule", {
-  value: true
+  value: true,
```

Comment:
I would suggest adding pre-commit hooks running Prettier in every module

Reply:
Added Husky for pre-commit hooks, will be added shortly. Should note though that this instance in particular is a part of dist/ which is ignored by prettier.